backend/models/Story.js

```diff
@@ -0,0 +1,85 @@
+module.exports = function(sequelize, Sequelize) {
+  return sequelize.define(
+    "stories",
+    {
+      id: {
+        type: Sequelize.INTEGER,
+        autoIncrement: true,
+        primaryKey: true
+      },
+      perspective: {
+        type: Sequelize.STRING
+      },
+      age: {
+        type: Sequelize.INTEGER
+      },
+      lgbtq: {
+        type: Sequelize.STRING
+      },
+      race: {
+        type: Sequelize.STRING
+      },
+      phone: {
+        type: Sequelize.STRING
+      },
+      topic: {
+        type: Sequelize.STRING
+      },
+      title: {
+        type: Sequelize.STRING
+      },
+      lede: {
+        type: Sequelize.STRING
+      },
+      published: {
+        type: Sequelize.DATE,
+        allowNull: true
+      },
+      like: {
+        type: Sequelize.INTEGER
+      },
+      cringey: {
+        type: Sequelize.INTEGER
+      },
+      haha: {
+        type: Sequelize.INTEGER
+      },
+      "me too": {
+        type: Sequelize.INTEGER
+      },
+      interesting: {
+        type: Sequelize.INTEGER
+      },
+      "story texts": {
+        type: Sequelize.STRING(5000)
```

Comment:
Is this an arbitrary length restriction or a current one? Should the other String fields also be similarly limited?

Reply:
some story texts at least 3000, gave a bit extra space